components/kyma-environment-broker/common/director/client.go

```diff
@@ -191,18 +175,9 @@ func (dc *Client) getRuntimeIdFromDirector(req *machineGraph.Request) (*getRunti
 	var success bool
 
 	for i := 0; i < reqAttempt; i++ {
```

Comment:
`for i := 0; i < reqAttempt; i++` is not necessary anymore, it was used to check X times in case token expire. Now oAuth client should handle many checks if token expire. Please check other clients.